internal/util/mqclient/pulsar_consumer.go

```diff
@@ -105,7 +106,15 @@ func (pc *PulsarConsumer) Ack(message Message) {
 // Close the consumer and stop the broker to push more messages
 func (pc *PulsarConsumer) Close() {
 	pc.c.Close()
```

Comment:
unsubscribe closed consumer will always failed, should unsub before close consumer